setup.sh

```diff
@@ -0,0 +1,5 @@
+git add .
```

Comment:
also even if you were deploying to heroku in the script, I don't understand why you'd make commits inside the script.